person_settings.xml

```diff
@@ -1,28 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
     <servers>
         <server>
-            <id>callisto-jparest</id>
-            <configuration>
-                <httpHeaders>
-                    <property>
-                        <name>Authorization</name>
-                        <value>Bearer ${ARTIFACTORY_TOKEN}</value>
-                    </property>
-                </httpHeaders>
-            </configuration>
+            <id>github-packages</id>
+            <!-- Any username would work as long as it's not empty -->
+            <username>DummyUser</username>
+            <password>${env.GITHUB_TOKEN}</password>
         </server>
     </servers>
 
     <profiles>
         <profile>
-            <id>artifactory</id>
+            <id>github</id>
             <repositories>
                 <repository>
-                    <id>callisto-jparest</id>
-                    <name>Callisto Jparest Repo</name>
-                    <url>https://artifactory.digital.homeoffice.gov.uk/artifactory/callisto-jparest</url>
+                    <id>github-packages</id>
```

Comment:
I am surprised the 2 repositories having the same id didn't cause any problem

Reply:
It is the ID of the server. See Maven docs for server
"id: This is the ID of the server (not of the user to login as) that matches the id element of the repository/mirror that Maven tries to connect to."

https://maven.apache.org/settings.html

Reply:
The excerpt you referred to is for the server element definition.
The id under repository element is the ID for the repository:
https://maven.apache.org/ref/3.9.2/maven-settings/settings.html#class_repository

It doesn't seem to break things. but it's just unexpected that the id element isn't fulfilling its implied function of identification